### PR TITLE
chore: add detailed error message for wasi uploads

### PR DESCRIPTION
**Pull request by Jhaines1988**
This PR adds a check for  WASI related errors and links users to the github issue about WASI support.

### CODE REVIEW

File: src/components/Errors/Alert.tsx [hunk 1]
```diff
@@ -9,6 +9,52 @@ const Alert: React.FC<AlertProps> = ({ message, dispatch }) => {
   const dismissError = () => {
     dispatch({ type: 'DISMISS_ERROR', payload: null });
   };
+
+  const renderMessage = () => {
+    return message === `Sorry! We currently don't support WASI.` ? (
```
Comment (nilslice) [inline src/components/Errors/Alert.tsx]: Could we specify further to say that `Sorry, WASI is currently not supported in the Playground. For now, use the Extism CLI to test WASI plug-ins.`

**Comment by nilslice** [comment]
@Jhaines1988 do you have a plugin that causes this error to be caught? I am trying with a WASI one in the [preview link](https://wasi-error.playground-1xh.pages.dev/), but still see: 

image

**Comment by Jhaines1988** [comment]
@nilslice  had to fix a typo. This should hopefully catch

File: src/components/Errors/Alert.tsx (hunk 1 above)
Comment (Jhaines1988) [reply src/components/Errors/Alert.tsx]: Sure, will do.

**Comment by nilslice** [comment]
> This should hopefully catch

Indeed it did! Checked again and works great

**Review by nilslice — APPROVED** [review]
:shipit:

**Pull request merged**